### DISCUSSION TITLE
0625: 更新最新版代码，gpt-4仍然报错403

### DISCUSSION THREAD

**Discussion by zhanzushun** [linweiyuan/go-chatgpt-api#189] [General]
OpenAI: {"detail":"Our systems have detected unusual activity from your system. Please try again later."} (code: 403)

**Comment by linweiyuan** [comment]
有没有配置 `GO_CHATGPT_API_ARKOSE_TOKEN_URL`，如果配置了还不行，那么暂时也没找到好的解决方法

你可以按照文章最下面的视频里，那样尝试去调试下，或者分享你的账号给我测试

**Reply by zhanzushun** [reply]
没看仔细文档，加了就好了。
原来你把 ARKOSE TOKEN 生成代理出去了，这样镜像更加稳定，赞！
